components/stacks-network/src/orchestrator.rs

```diff
@@ -151,7 +160,29 @@ impl DevnetOrchestrator {
         })
     }
 
-    pub async fn prepare_network(&mut self) -> Result<ServicesMapHosts, String> {
+    pub fn prepare_network_k8s_coordinator(
+        &mut self,
+        namespace: &str,
+    ) -> Result<ServicesMapHosts, String> {
+        let services_map_hosts = ServicesMapHosts {
+            bitcoin_node_host: format!(
+                "bitcoind-chain-coordinator-service.{namespace}.svc.cluster.local:18443"
+            ),
+            stacks_node_host: format!("stacks-node-service.{namespace}.svc.cluster.local:20443"),
+            postgres_host: format!("stacks-api-service.{namespace}.svc.cluster.local:5432"),
+            stacks_api_host: format!("stacks-api-service.{namespace}.svc.cluster.local:3999"),
+            stacks_explorer_host: "localhost".into(), // todo (micaiah)
+            bitcoin_explorer_host: "localhost".into(), // todo (micaiah)
+            subnet_node_host: "localhost".into(),     // todo (micaiah)
+            subnet_api_host: "localhost".into(),      // todo (micaiah)
```

Comment:
Could we please create an issue for tracking these todos? Thanks!

Reply:
Done: https://github.com/hirosystems/clarinet/issues/1113